packages/mongo-nav/src/MongoNav.spec.tsx

```diff
@@ -417,4 +417,27 @@ describe('packages/mongo-nav', () => {
       });
     });
   });
+
+  describe('when the date is before MongoDB World', () => {
+    test('getProductHost function returns stitch', () => {
+      expect(getProductHost()).toBe('stitch');
+    });
+  });
+
+  describe('when the date is the day of MongoDB World', () => {
+    const mdbworld = new Date('June 8, 2020 0:00:00');
+
+    test('getProductHost function returns realm', () => {
+      const productName = getProductHost(mdbworld);
```

Comment:
Testing this like so doesn't seem to actually test that our implementation is correct. We don't pass any parameters to the function, so we should be testing that the function returns the correct name when nothing is passed.

Reply:
We do that on line 422.